docs/plugins/main.py

```diff
@@ -111,31 +111,33 @@ def install_logfire(markdown: str, page: Page) -> str:
         # Split them and strip quotes for each one separately.
         extras = [arg.strip('\'"') for arg in arguments[1].strip('[]').split(',')] if len(arguments) > 1 else []
         package = 'logfire' if not extras else f"'logfire[{','.join(extras)}]'"
-        extras_arg = ' '.join(f'-E {extra}' for extra in extras)
+        # Hiding unused arg because the linter is yelling at me
+        #  extras_arg = ' '.join(f'-E {extra}' for extra in extras)
```

Comment:
This produces the wrong output:

<img width="717" alt="Screenshot 2024-09-25 at 12 01 01" src="https://github.com/user-attachments/assets/c188752c-951c-45f6-8b4c-752ad1865cfd">

Reply:
Thanks @Kludex for this pick up - I forgot to chase this one up, I got blocked trying to make a different change to the `main.py` file,  because of this error on commit: 

![Screenshot 2024-09-25 at 13 02 43](https://github.com/user-attachments/assets/47412c36-536d-44d0-b84c-b3c7bee18dba)

NGL I don't really understand what's going on here. Can you help?